README.md

```diff
@@ -1,33 +1,93 @@
 # PiCraft
 
-### Control a Minecraft world on a Raspberry Pi from Julia
+### Control a Minecraft world on the Minecraft: Pi or Java edition from Julia
 
 ## Installation and Usage
+
+### Get Minecraft
+
+- Get the Minecraft: Pi edition [here](https://minecraft.net/en-us/edition/pi/). It's free!
+- Get the Minecraft: Java edition(MacOS, Linux and Windows) [here](https://minecraft.net/en-us/download/alternative).
+
+### Get RaspberryJuice or RaspberryJamMod (for Minecraft: Java edition)
+
+Officially the ability to communicate with the Minecraft world is only available for the Minecraft: Pi edition. To get this to work on the Java edition we need to install a Mod.
+
+- "RaspberryJamMod" is a Forge Mod, if you wish to use this along with other Forge mods then this is recommended. Installation instructions are available [here](http://www.instructables.com/id/Python-coding-for-Minecraft/).
+- "RaspberryJuice" is a Bukkit server plugin, recommended if you wish to work on a Bukkit Minecraft server. Get it [here](https://dev.bukkit.org/projects/raspberryjuice).
+
+There is a disagreement in the coordinates reported by Minecraft's debug mode and the `player.getPos()` command. Enter `/setworldspawn 0 0 0` in the Minecraft console to remove this discrapency.
```

Comment:
Is this true of both Jam and Juice? Would be good to have a sentence here about the underlying cause... `Minecraft Pi sets the spawn point as (0,0,0) but the java version......` etc...

Reply:
True for both. Although there is a setting available in Juice to change b/w absolute and relative coordinate systems. I purposely cut it out. I don't want to increase this document's length at this point. Writing about camera angles, directions and some tips and tricks are at a higher priority but I want to delay this work until I get a better understanding of this project.